### PR TITLE
feat: add support for additional pod annotations/labels

### PR DESCRIPTION
**Pull request by jaygridley**

### CODE REVIEW

**Comment by jaygridley** [comment]
Hello @JoshVanL, could you have a look at this one, please?

**Comment by JoshVanL** [comment]
Thanks @jaygridley!
/lgtm
/approve 
/ok-to-test

**Comment by JoshVanL** [comment]
CI broken?
/retest

**Comment by jaygridley** [comment]
Hi @JoshVanL, I did that, but the tests still seem to struggle to run.

**Comment by jaygridley** [comment]
@JoshVanL I think this is ready to go

File: deploy/charts/trust-manager/values.yaml [hunk 1]
```diff
@@ -108,6 +108,12 @@ app:
     # -- If false, disables the default seccomp profile, which might be required to run on certain platforms
     seccompProfileEnabled: true
 
+  # -- Pod labels to add to trust pods.
```
Comment (erikgb) [inline deploy/charts/trust-manager/values.yaml:111]:
```suggestion
  # -- Pod labels to add to trust-manager pods.
```

File: deploy/charts/trust-manager/values.yaml [hunk 2]
```diff
@@ -108,6 +108,12 @@ app:
...
+  podLabels: {}
+
+  # -- Pod annotations to add to trust pods.
```
Comment (erikgb) [inline deploy/charts/trust-manager/values.yaml:114]:
```suggestion
  # -- Pod annotations to add to trust-manager pods.
```

**Review by erikgb — CHANGES_REQUESTED** [review]
Thanks! A couple of minor nits, but that might be because the project has been renamed since the PR was created. Sorry for the delay.

**Comment by erikgb** [comment]
@jaygridley Maybe you should rebase/squash all commits into a single one? And regen the Helm docs.

**Comment by JoshVanL** [comment]
Cheers! 
/lgtm

**Pull request merged**